apps/geo/serializers.py

```diff
@@ -64,6 +64,11 @@ def validate_project(self, project):
             raise serializers.ValidationError('Invalid project')
         return project.id
 
+    def validate(self, data):
+        if self.instance and self.instance.published == True:
```

Comment:
```suggestion
        if self.instance and self.instance.is_published:
```

Also, add `is_published` to Meta.exclude. And add a new endpoint to publish the region.